src/slackeros/ros_connector.py

```diff
@@ -95,6 +103,47 @@ def __init__(
         self.queue_worker.setDaemon(True)
         self.queue_worker.start()
 
+    def log_image(self, type=None, topic='/head_xtion/rgb/image_color'):
+        class ImageUploader(Thread):
+
+            def __init__(self, access_token, bridge, send_image, type=None, topic='/head_xtion/rgb/image_color'):
+                Thread.__init__(self)
+                self.access_token = access_token
+                self.bridge = bridge
+                self.send_image = send_image
+                self.type = type
+                self.topic = topic
+
+            def run(self):
+                # upload image
+                image = rospy.wait_for_message(self.topic, Image, timeout=1.5)
```

Comment:
what happens if the image is not received in 1.5 seconds, here? Shouldn't catch that exception?